overview/http_server.md

```diff
@@ -3,14 +3,16 @@
 A slightly more interesting example is an HTTP Server:
 
 ```crystal
+
 require "http/server"
 
-server = HTTP::Server.new(8080) do |context|
-  context.response.content_type = "text/plain"
-  context.response.print "Hello world! The time is #{Time.now}"
+server = HTTP::Server.new do |context|
+  context.response.headers["Content-Type"] = "text/plain"
+  context.response.print("Hello world! The time is #{Time.now}")
```

Comment:
ditto

Reply:
This was more idiomatic as-is. That example is pretty old, and I think it predates `HTTP::Response#content_type`. The example was fine as-is anyway, no need to inflate the diff.